card.rb

```diff
@@ -0,0 +1,14 @@
+# Class structure for card, it has a numeric value and a suit
+class Card
+  attr_reader :val, :suit
+  attr_accessor :val, :suit
+
+  def initialize(val, suit)
+    @val = val
```

Comment:
is `val` the same as `num` in the other classes? If they're the same I think it makes sense to be consistent across the board.